php/php-xhprof/Portfile

```diff
@@ -1,47 +1,52 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem              1.0
+PortGroup               github 1.0
 PortGroup               php 1.1
 
 name                    php-xhprof
-version                 0.9.4
+version                 2.3.7
 categories-append       devel
 platforms               darwin
-maintainers             ryandesign openmaintainer
+maintainers             {ryandesign @ryandesign} openmaintainer
 license                 Apache-2
 
-php.branches            5.3 5.4 5.5 5.6
+php.branches            7.0 7.1 7.2 7.3 7.4 8.0 8.1
 php.pecl                yes
-php.pecl.prerelease     yes
 
 description             A Hierarchical Profiler for PHP
 
 long_description        XHProf is a function-level hierarchical profiler for \
                         PHP and has a simple HTML based navigational interface.
 
-checksums               rmd160  e484c4902f287ef18d9f75dda75a28d73bb8272d \
-                        sha256  002a2d4a825d16aeb3017c59f94d8c5e5d06611dd6197acd2f07fce197d3b8f8
+checksums               rmd160  f6fa04037e0de6feb73cda777c4dce7348c29985 \
+                        sha256  378dd13f29185f8e5e9c0cc81e360e1ed71a4ac3eb3cc4e8d85e732da0300426 \
+                        size    842891
 
 if {${name} ne ${subport}} {
     patchfiles          patch-callgraph_utils.php.diff
-    
+
     post-patch {
         reinplace "s|@PREFIX@|${prefix}|g" ${worksrcpath}/xhprof_lib/utils/callgraph_utils.php
     }
-    
+
     php.build_dirs      ${worksrcpath}/extension
-    
+
     use_parallel_build  yes
-    
+
     post-destroot {
         # Install php files not installed by the Makefile
         xinstall -d ${destroot}${prefix}/www/${subport}
         copy ${worksrcpath}/examples ${destroot}${prefix}/www/${subport}/examples
         copy ${worksrcpath}/xhprof_lib ${destroot}${prefix}/www/${subport}/xhprof_lib
         copy ${worksrcpath}/xhprof_html ${destroot}${prefix}/www/${subport}/xhprof_html
     }
-    
+
     variant graphviz description {Build with graphviz, support for callgraphs} {
         depends_run-append      path:bin/dot:graphviz
     }
 }
+
+if {${name} ne ${subport}} {
```

Comment:
A second `if {${name} ne ${subport}}` block isn't needed; this can be combined with the first one.

Reply:
Applied locally, will be part of next push, thanks!